ci/jobs/pulp-dev.yaml

```diff
@@ -89,12 +89,18 @@
             rm -rf logs
             mkdir logs
             if [ "$(which journalctl 2>/dev/null)" ]; then
-                journalctl > logs/syslog
+                sudo journalctl > logs/syslog
             else
-                cat /var/log/messages > logs/syslog
+                if [ -f /var/log/messages ]; then
+                    sudo cat /var/log/messages > logs/syslog
+                else
+                    echo "Failed to get syslogs journalctl neither" \
+                    "/var/log/messages was not found" > logs/syslogs
```

Comment:
How about rewording this to "Failed to get syslogs: neither journalctl nor /var/log/messages were found."